### PR TITLE
RS 6.2.14 release note preview

### PR DESCRIPTION
**Pull request by lanceleonard**
- Initial draft

### CODE REVIEW

File: content/rs/release-notes/rs-6-2-14.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+{{<image filename="images/rs/cluster-settings-preferences.png" alt="Preferences settings in the Redis Software admin console." width="75%">}}{{< /image >}}
+
+## Version changes 
+
+### Prerequisites and notes 
+
+-  You can [upgrade to v6.2.14](https://docs.redis.com/latest/rs/installing-upgrading/upgrading/) from Redis Enterprise Software v6.2 and later. 
+
+- The database upgrade process is controlled by the `redis_upgrade_policy` policy setting.  To learn more, see the [Database upgrade default changes](https://docs.redis.com/latest/rs/release-notes/rs-6-2-4-august-2021/#database-upgrade-default-changes) section of the [v6.2.4 release notes](https://docs.redis.com/latest/rs/release-notes/rs-6-2-4-august-2021/).
```
Comment (rrelledge) [inline content/rs/release-notes/rs-6-2-14.md:72]: Maybe remove "policy" here?
```suggestion
- The database upgrade process is controlled by the `redis_upgrade_policy` setting.  To learn more, see the [Database upgrade default changes](https://docs.redis.com/latest/rs/release-notes/rs-6-2-4-august-2021/#database-upgrade-default-changes) section of the [v6.2.4 release notes](https://docs.redis.com/latest/rs/release-notes/rs-6-2-4-august-2021/). 
```

File: content/rs/release-notes/rs-6-2-14.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+### Prerequisites and notes 
+
+-  You can [upgrade to v6.2.14](https://docs.redis.com/latest/rs/installing-upgrading/upgrading/) from Redis Enterprise Software v6.2 and later. 
+
+- The database upgrade process is controlled by the `redis_upgrade_policy` policy setting.  To learn more, see the [Database upgrade default changes](https://docs.redis.com/latest/rs/release-notes/rs-6-2-4-august-2021/#database-upgrade-default-changes) section of the [v6.2.4 release notes](https://docs.redis.com/latest/rs/release-notes/rs-6-2-4-august-2021/). 
+
+- Upgrades from versions earlier than v6.2 are not supported.
+
+- If you are using Active-Active or Active-Passive (ReplicaOf) databases and experience synchronization issues as a result of the upgrade, see RS67434 details in [Resolved issues](#resolved-issues) for help resolving the problem.
```
Comment (rrelledge) [inline content/rs/release-notes/rs-6-2-14.md:76]:
```suggestion
- If you are using Active-Active or Active-Passive (Replica Of) databases and experience synchronization issues as a result of the upgrade, see RS67434 details in [Resolved issues](#resolved-issues) for help resolving the problem.
```

File: content/rs/release-notes/rs-6-2-14.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+## Resolved issues 
+
+- RS38320 A failed task leaves the nodes list outdated in the UI
+- RS69256 Change pre-bootstrap default TLS versions to 1.2
+- RS67133 Fixed replication for command RESTOREMODAUX 
+- RS66468 Fixed “Test regex keys” in the UI
+- RS58156 Fixed the installation to abort and alert when encountering issues in NTP
+- RS67935 Fixed releasing 30MB of memory when deleting an Active-Active database
+- RS64276 Fixed high memory consumption of the DMC output buffers when running CLIENT LIST
+- RS54131 Fixed returning +OK reply from the QUIT command on a TLS enabled database
```
Comment (rrelledge) [inline content/rs/release-notes/rs-6-2-14.md:112]:
```suggestion
- RS54131 Fixed returning +OK reply from the QUIT command on a TLS-enabled database
```

File: content/rs/release-notes/rs-6-2-14.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+- RS55248 - Added support for the MODULE LIST command on Active-Active databases
+- RS64316 - Enhanced validity checks of the input parameters of the CRDB-CLI tool
+
+## Resolved issues 
+
+- RS38320 A failed task leaves the nodes list outdated in the UI
+- RS69256 Change pre-bootstrap default TLS versions to 1.2
+- RS67133 Fixed replication for command RESTOREMODAUX 
+- RS66468 Fixed “Test regex keys” in the UI
+- RS58156 Fixed the installation to abort and alert when encountering issues in NTP
```
Comment (rrelledge) [inline content/rs/release-notes/rs-6-2-14.md:109]:
```suggestion
- RS58156 Fixed the installation to stop and alert when encountering issues in NTP
```

File: content/rs/release-notes/rs-6-2-14.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+- RS67935 Fixed releasing 30MB of memory when deleting an Active-Active database
+- RS64276 Fixed high memory consumption of the DMC output buffers when running CLIENT LIST
+- RS54131 Fixed returning +OK reply from the QUIT command on a TLS enabled database
+
+## Security
+
+### Open source Redis security compatibility
+
+As part of Redis commitment to security, Redis Enterprise Software implements the latest [security fixes](https://github.com/redis/redis/releases) available with open source Redis.
```
Comment (rrelledge) [inline content/rs/release-notes/rs-6-2-14.md:118]: Or maybe reword slightly?
```suggestion
As part of Redis's commitment to security, Redis Enterprise Software implements the latest [security fixes](https://github.com/redis/redis/releases) available with open source Redis. 
```

**Review by rrelledge — APPROVED** [review]
Looks good to me. I just left some small suggestions.

**Comment by lanceleonard** [comment]
Closing this PR in favor of PR #2156, which replaces this one.
